react/Header/UserAccount/UserAccount.js

```diff
@@ -46,7 +51,10 @@ export default class UserAccount extends Component {
   }
 
   handleMenuToggleClick = () => {
-    this.setState(state => ({ menuOpen: !state.menuOpen }));
+    this.setState(state => {
+      this.props.menuToggleCallBackFunction(!state.menuOpen);
```

Comment:
Just to future proof the API a bit, it might be a good idea to pass an object to the callback so that we could add more stuff later without it being a breaking change. My vote is for `{ open: !state.menuOpen }`, since it seems close to what a native DOM API might look like.